packages/web-twig/src/Resources/components/FileUploader/FileUploaderAttachment.twig

```diff
@@ -19,7 +19,8 @@
 
 {# Attributes #}
 {%- set _dataImagePreviewAttr = _generateImagePreview ? 'data-spirit-imagePreview="true"' : null -%}
-{%- set _imageObjectFitAttr = _imageObjectFit ? 'data-spirit-image-object-fit=' ~ _imageObjectFit : null -%}
+{%- set _imageObjectFitCSSAttr = '--file-uploader-attachment-image-object-fit:' -%}
+{%- set _imageObjectFitAttr = _imageObjectFit ? 'style=' ~ _imageObjectFitCSSAttr ~ _imageObjectFit : null -%}
```

Comment:
The `_imageObjectFitCSSAttr` variable is not necessary. This is how we deal with unwanted escaping in our Twig components:

```suggestion
{%- set _imageObjectFitAttr = _imageObjectFit ? 'style="--file-uploader-attachment-image-object-fit: ' ~ _imageObjectFit | escape('html_attr') ~ '"' : null -%}
```

Then you need to call the `raw` filter on it and it's done:

```twig
{{ _imageObjectFitAttr | raw }}
```

Reply:
Thank you! Fixed 🪄